3_visualize/src/merge_plot_legend.R

```diff
@@ -16,14 +16,21 @@ merge_plot_legend <- function(main_plot, legend){
     draw_plot(legend_updated, x = 0.83, y = 0.1, width = 0.08, height = 0.8)+
     # and annotate temperature threshold
     draw_label(expression('75'~degree*'F'), colour = "orangered", x = .95, y = .75, size = 8)+
-    annotate("segment", xend = 0.903, x = 0.92, yend = 0.62, y = 0.738, 
+    annotate("segment", xend = 0.903, x = 0.92, yend = 0.62, y = 0.69, 
              arrow = arrow(length = unit(0.1, "cm")), colour = "orangered")+
     draw_label("threshold", x = .96, y = .705, size = 6, colour = "orangered")+
     # and annotate mean
-    draw_label("Mean", colour = "darkgreen", x = 0.95, y = 0.6, size = 8)+
-    annotate("segment", xend = 0.903, x = 0.92, yend = 0.56, y = 0.56, 
-             arrow = arrow(length = unit(0.1, "cm")), colour = "darkgreen")
+    draw_label("Mean", colour = "darkgreen", x = 0.95, y = 0.545, size = 7)+
+    annotate("segment", xend = 0.903, x = 0.92, yend = 0.545, y = 0.545, 
+             arrow = arrow(length = unit(0.1, "cm")), colour = "darkgreen")+
+    # and 90% CI
+    draw_label("{", x = 0.9, y = 0.47, size = 20, angle = 180, color = "cadetblue3")+
+    draw_label("+/- 90% CI", colour = "cadetblue3", x = 0.95, y = 0.47, size = 7)+
```

Comment:
```suggestion
    draw_label("\u00B1  90% CI", colour = "cadetblue3", x = 0.95, y = 0.47, size = 7)+
```

Reply:
The suggested edit will use a unicode character for +/-

Reply:
Got it! I meant to look it up, thanks for doing that!